envoy/http/filter.h

```diff
@@ -174,11 +174,21 @@ enum class FilterTrailersStatus {
 
 /**
  * Return codes for encode metadata filter invocations. Metadata currently can not stop filter
- * iteration.
+ * iteration except in the case of local replies.
  */
 enum class FilterMetadataStatus {
-  // Continue filter chain iteration.
+  // Continue filter chain iteration for metadata only. Does not unblock returns of StopIteration*
+  // from  (decode|encode)(Headers|Data).
```

Comment:
nit: two spaces between "from" and "(decode|"

Reply:
Done.